.gitlab/package_build/deb.yml

```
@@ -19,6 +19,10 @@
     - chmod 0744 /tmp/system-probe/clang-bpf /tmp/system-probe/llc-bpf
     - inv -e omnibus.build --release-version "$RELEASE_VERSION" --major-version "$AGENT_MAJOR_VERSION" --python-runtimes "$PYTHON_RUNTIMES" --base-dir $OMNIBUS_BASE_DIR ${USE_S3_CACHING} --skip-deps --go-mod-cache="$GOPATH/pkg/mod" --system-probe-bin=/tmp/system-probe ${FLAVOR:+--flavor $FLAVOR}
     - ls -la $OMNIBUS_PACKAGE_DIR
+    - curl -sSL "https://dd-package-tools.s3.amazonaws.com/dd-pkg/${DD_PKG_VERSION}/dd-pkg_Linux_${DD_PKG_ARCH}.tar.gz" | tar -xz -C /usr/local/bin dd-pkg
+    - pushd $OMNIBUS_PACKAGE_DIR
+    - find . -iregex '.*\.\(deb\|rpm\)' -printf '%f\0' | xargs -0 dd-pkg lint
+    - popd
```

Could we factorise this with a `!reference`, wdyt?
```suggestion
    - !reference [.lint_linux_packages]
```

---

Yeah, that's a fine suggestion - I was hoping to be able to provide a GitLab template do handle the linting, but I don't think our building process is consistent enough between products to have that make sense.

Depending on the outcome with Hugo's suggestion I'll either make this a reference or a shared bash script to reduce duplication.